models/completed_data_maps.go

```diff
@@ -0,0 +1,64 @@
+package models
+
+import (
+	"encoding/json"
+	"time"
+	"github.com/gobuffalo/pop"
+	"github.com/gobuffalo/validate"
+	"github.com/gobuffalo/validate/validators"
+)
+
+type CompletedDataMap struct {
+	ID          int       `json:"id" db:"id"`
```

Comment:
You can merge fields from another struct like this:
```go
type CompletedDataMap struct {
    models.DataMap
}
```

Reply:
This doesn't seem to work, or at least I can't seem to make it work.  You can inherit from other structs, but the fields won't keep the same indentation level.  This demonstrates the issue:  https://play.golang.org/p/8VH15cNga9R  

Unless there is some other way to do it.

Reply:
ok, nvm then